multichain-testing/tools/query.ts

```diff
@@ -10,8 +10,15 @@ import type { QueryDenomHashResponseSDKType } from '@agoric/cosmic-proto/ibc/app
 import type { QueryChannelResponseSDKType } from '@agoric/cosmic-proto/ibc/core/channel/v1/query.js';
 import type { QueryChannelsResponseSDKType } from '@agoric/cosmic-proto/ibc/core/channel/v1/query.js';
 
+// TODO: export tendermint Block from @agoric/cosmic-proto
```

Comment:
@0xpatrickdev you suggested

```ts
import type { BlockSDKType } from '@agoric/cosmic-proto/tendermint/types/block.js';
```

but that type doesn't match; for `block.header.time`, it has `{    seconds: bigint;    nanos: number; }` but what I observe is a `string`.